joomla/joomla-cms
# This module shows a list of the currently Logged-in Users.

This module shows a list of the currently Logged-in Users.
So giving it a title of LAST Logged-in Users is incorrect as that would be a list of the last x users who have logged in and might not be logged in anymore.

### Summary of Changes
Changes the language strings to more accurately describe what the module does.

### Testing Instructions
* Create an admin module of type "Logged-in Users" and assign it to the cpanel position (or use the existing one)
* Enable the "Automatic Title" module option (in Advanced tab)
* Check the title of the module in the Control Panel
* Change the "Count" module option and verify that the module title adjusts:
   "0" should give "Logged-in Users"
   "1" should give "Logged-in User"
   "5" should give "5 Logged-in Users"

#### Disclaimer
Work is done by @brianteeman
